include/veriblock/pop/storage/util.hpp

```diff
@@ -113,7 +113,7 @@ void saveTree(
   std::vector<const index_t*> dirty_indices;
 
   // map pair<hash, shared_ptr<index_t>> to vector<index_t*>
-  for (auto& index : tree.getAllBlocks()) {
+  for (auto& index : tree.getBlocks(false)) {
```

Comment:
I think it would be better to have a enum class 
```
enum class GetBlocksPolicy {
  DEFAULT,
  SKIP_DELETED
}

```

and set getBlocks(GetBlocksPolicy policy = GetBlocksPolicy::DEFAULT)

Reply:
A bit too verbose but I can do it this way.

Reply:
Done